qabelbox/src/main/java/de/qabel/qabelbox/helper/AccountHelper.java

```diff
@@ -19,7 +19,7 @@
     public static final Account DEFAULT_ACCOUNT = new Account(ACCOUNT, ACCOUNT_TYPE);
     public static final String AUTHORITY = BuildConfig.AUTHORITY;
     // every 3 hours
-    public static long SYNC_INTERVAL = 60*60*3;
+    public static long SYNC_INTERVAL = 60*30;
```

Comment:
Why the change?

Also, if you change the interval you may update or remove the explaining comment